autoload/jedi.vim

```diff
@@ -489,6 +489,8 @@ function! jedi#complete_opened(is_popup_on_dot)
         " option is set.
         if g:jedi#popup_select_first && stridx(&completeopt, 'longest') > -1
             return "\<Down>"
+        else
+            return ""
```

Comment:
@davidhalter Can you clarify how it breaks it? In my hands all combinations of ``popup_on_dot`` and ``popup_select_first`` work as expected. If you can tell me how you think it will cause an issue, I will be very happy to fix it.